storybook/stories/semantic-ui/react/dropdown/standard.stories.mdx

```diff
@@ -256,14 +256,15 @@ A dropdown can take the width of its container.
 
 A dropdown can vary in size.
 
-Available sizes are: "small" and "medium".
+Available sizes are: `small`, `medium` and `large`.
```

Comment:
Change for:

"Can be set to: "small", "large" or omitted." to be consistant with the other pages.

![image](https://user-images.githubusercontent.com/794579/74883012-11353980-533e-11ea-8b16-0bec256e4423.png)